client/app/views/templates/_view_step_base.jade

```diff
@@ -1,5 +1,5 @@
 header
-    figure
+    figure(data-type=stepName)
```

Comment:
I prefer to avoid a `data-type` attribute, or, if we really need it, to use an `id` instead (which, in fact, is what it is here).